src/mjolnir/pbfgraphparser.cc

```diff
@@ -1226,7 +1231,12 @@ struct graph_callback : public OSMPBF::Callback {
       tag_ = kv;
       const auto it = tag_handlers_.find(tag_.first);
       if (it != tag_handlers_.end()) {
-        it->second();
+        try {
+          it->second();
+        } catch (const std::exception& ex) {
+          LOG_ERROR("Error during parsing of `" + tag_.first + "` tag: " + std::string{ex.what()});
```

Comment:
we should log the wayid and the value so that when this happens we can make the parsing robust to it or eventually support new tagging patterns. i would switch it to a WARN rather than an ERROR though.